Do not fail if related usns fail (SC-1300) (#2497)
canonical/ubuntu-pro-client: docs/references/what_are_related_usns.md

# What are related USNs

> A [USN](what_is_a_usn.md) is composed by different CVEs. If a CVE appears on multiple USNs, we say

Reviewer: ```suggestion
An [Ubuntu Security Notice (USN)](what_is_a_usn.md) is composed of different Common Vulnerabilities and Exposures (CVEs). If a CVE appears on multiple USNs, we say
```

that those USNs are related. On the following image, we can see visual representation of that

> concept:

Reviewer: ```suggestion
concept, where USN-789 and USN-321 are related USNs because both are affected by CVE-2:
```

> ![Related USN example](../images/usn-related.png)

Reviewer: I really like the diagram, I think it shows the hierarchy really clearly, but would it be possible to change the colours used? A significant chunk of humans are red/green colourblind so using those colours together is usually better avoided.

Author: Good point, I can change that

A real example can be seen on [USN-5573-1](https://ubuntu.com/security/notices/USN-5573-1),
that express in the section **Related notices** that both **USN-5570-1** and **USN-5570-2**
are related to **USN-5573-1**.

> This is information can useful for users that want to tackle

Reviewer: ```suggestion
This information is useful for users that want to tackle
```

> all related USNs at once, making sure that a CVE is indeed fixed on their Ubuntu machine.

Reviewer: ```suggestion
all related USNs at once, making sure that a CVE is fully fixed on their Ubuntu machine.
```
